static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundle-digital-sub-price-test-1.js

```diff
@@ -4,128 +4,81 @@ define([
-                }
-            }
-        };
+    var buildTemplate = function(variant) {
+        return template(contributionsEpicSingleButton, {
+            linkUrl1: config.page.membershipUrl + '/bundles?INTCMP=' + 'BUNDLE_PRICE_TEST_1M_' + config.page.edition.toUpperCase() + '_' + variant.id.toUpperCase(),
+            componentName: variant.componentName,
+            title: 'Since you’re here…',
+            p1: '… we’ve got a small favour to ask. More people are reading the Guardian than ever, but far fewer are paying for it. Advertising revenues across the media are falling fast. And ' +
+            '<span class="contributions__highlight">unlike many news organisations, we haven’t put up a paywall – we want to keep our journalism as open as we can</span>' +
+            '. So you can see why we need to ask for your help. The Guardian’s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters – because it might well be your perspective, too.',
+            p2: 'If everyone who reads our reporting, who likes it, helps to support it, our future would be much more secure.',
+            p3: '',
+            cta1: 'Support the Guardian'
+        });
+    };
 
-        this.showThrasher = function() {
-            var thrasherContainer = this.thrasherContainer();
-            if (thrasherContainer) {
-                if (!thrasherContainer.classList.contains('visible')) {
-                    thrasherContainer.classList.add('visible');
-                }
-            }
-        };
+    return contributionsUtilities.makeABTest({
+        id: 'BundleDigitalSubPriceTest1M',
+        campaignPrefix: '',
+        campaignSuffix: '',
+        start: '2017-05-10',
+        expiry: '2017-07-06',
 
-        this.setup = function(variant) {
-            this.setTitle();
-            this.setCopy();
-            this.setLink(variant);
-            this.setCaption();
-            this.showThrasher();
-        };
+        author: 'Justin Pinner',
+        description: 'Test digital subs price points via epic',
+        successMeasure: '',
+        idealOutcome: 'Find the price that works for most people',
+        hypothesis: 'One of our price points will be more desirable than the others',
+        audienceCriteria: 'Non-paying UK edition readers - mobile resolution and above',
+        audience: 0.1,
+        audienceOffset: 0,
+        showForSensitive: false,
+        useTargetingTool: false,
 
-        this.completeFunc = function(complete) {
-            // fire on thrasher's [find out more -->] button click
-            bean.on(qwery('.membership-ab-thrasher--wrapper .link-button')[0], 'click', complete);
-        };
+        overrideCanRun: false,
+        canRun: function () {
+            return !cookies.getCookie('GU_DBPT1M') &&
+                !userFeatures.isPayingMember() &&
+                config.page.edition.toUpperCase() === 'UK';
```

Comment:
Or is it that you do want to show it to existing contributors but not members? If this is the case, why you want to do this? The number of existing contributors in comparison to the whole audience is a drop in the ocean, and the number of them who will convert will be too low (probably 0) to learn anything from anyway, so I would recommend sticking to the default paying member checks